src/views/profile/profileView.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import NavBar from "../../components/navigation/navBar";
+import PropTypes from "prop-types";
+import { Button } from "reactstrap";
+import decode from "jwt-decode";
+import { AUTHENTICATED } from "../../utils/myHeaders";
+
+const userProfile = props => {
+  let {
+    username,
+    image,
+    bio,
+    first_name,
+    last_name
+  } = props.profile;
+  let isOwner;
+  let loggedInUser = decode(AUTHENTICATED);
+  loggedInUser.username === username ? (isOwner = true) : (isOwner = false);
+
+  let bt = "";
+  if (isOwner) {
+    bt = (
+      <Button
+        color=""
+        className="EditLink"
+        id="EditProfile"
+        href={`/profile/edit/${username}`}
+      >
+        Edit Profile
+      </Button>
+    );
+  } else {
+    bt = (
+      <Button
+        color=""
+        className="EditLink"
+        id="FollowProfile"
+        onClick={props.handleFollow}
+      >
+        Follow
+      </Button>
+    );
+  }
+
+  return (
+    <>
```

Comment:
Use React.Fragment instead of open and close tags, see https://reactjs.org/docs/fragments.html, and remember to de-structure it at the top so as to simply use <Fragment> ...content ... </Fragment> . Implement this in all your components

Reply:
I have worked on it